### PR TITLE
Rm ddt

### PR DESCRIPTION
**Pull request by bradleybogg**
PR requested onbehalf of Renee Mateo.

Renee to modify description between asking for review.

### CODE REVIEW

**Comment by majore15** [comment]
Please ignore Update program from nmap to crunch.

**Review by Phaellynn — DISMISSED** [review]
Crunch amendments pass all functionality tests.
1. Tool saves generated passwords to a file without error
2. Tool generates large passwords without error
Screenshot 2023-12-14 100201

**Review by bradleybogg — COMMENTED** [review]
Please remove this file from your PR

**Comment by majore15** [comment]
@bradleybogg 
Hi Bradley.  I have completed the pull merge. Please review.  Thanks.

File: yarn.lock
Comment (Sakada-lim) [inline yarn.lock:1]: I have updated the Yarn.lock and fixed the conflicted issue on the merge request.

**Review by Sakada-lim — APPROVED** [review]
Conflicting issue is fix. Noted next time please only add the file that you work on. DO NOT use git add . if you don't know what you are doing

**Pull request merged**